[MPD] Add HTTP stream support

toam-plus opened the pull request:
Compile the MPD with HTTPD stream support and MP3 streams:
1. Add MP3 Encoder "lame" to "PKG_DEPENDS_TARGET"
2. --enable-lame-encoder at "PKG_CONFIGURE_OPTS_TARGET"
3. --enable-httpd-output at "PKG_CONFIGURE_OPTS_TARGET"

chewitt commented:
Your gravatar violates the terms of use for the Raspberry pi foundations logo (it should not be obscured by or combined with other marks) and ours looks like there's a ladybug crawling on it. PKG_REV needs to be bumped.

toam-plus commented:
Hello chewitt, thanks for the note with my Gravatar. I have changed the picture. :)

Furthermore, I've changed the "package.mk" file. The package revision is now "1".

chewitt commented:
rebase and squash to a single commit please
